src/main/java/com/example/kadai9/MovieController.java

```diff
@@ -35,8 +41,9 @@ public ResponseEntity<Map<String, String>> createMovie(@RequestBody CreateMovieF
         return ResponseEntity.created(url).body(Map.of("message", "the movie successfully created"));
     }
 
-    @PatchMapping("/movies")
-    public ResponseEntity<Map<String, String>> updateMovie(@RequestBody Movie movie) {
+    @PatchMapping("/movies/{id}")
+    public ResponseEntity<Map<String, String>> updateMovie(@PathVariable("id") int id, @RequestBody CreateMovieForm createMovieForm) {
```

Comment:
updateなので
```suggestion
    public ResponseEntity<Map<String, String>> updateMovie(@PathVariable("id") int id, @RequestBody Form form) {
```
か
```suggestion
    public ResponseEntity<Map<String, String>> updateMovie(@PathVariable("id") int id, @RequestBody UpdateForm form) {
```
としたいですね。

Reply:
CreateMovieForm -> MovieFormに変更しました。参照先も同じく変更
172518547843fcd8b9921fd02371373ab7b35fc3